recipes-bsp/tegra-binaries/tegra-redundant-boot_32.3.1.bb

```diff
@@ -50,12 +52,13 @@ PACKAGES = "${PN}-nvbootctrl ${PN} ${PN}-dev"
 FILES_${PN}-nvbootctrl = "${sbindir}/nvbootctrl"
 FILES_${PN} += "/opt/ota_package"
 FILES_${PN}-dev = "${datadir}/nv_tegra/rollback"
-SYSTEMD_SERVICE_${PN} = "nv_update_verifier.service"
+SYSTEMD_SERVICE_${PN} = "${@bb.utils.contains('DISTRO_FEATURES', 'systemd', 'nv_update_verifier.service', '', d)}"
```

Comment:
This shouldn't be necessary; systemd.bbclass should handle this for us.

Reply:
This is indeed not necessary. I checked it without the change and it build fine for sysvinit.